legacy/src/main/java/macrobase/ingest/CSVIngester.java

```diff
@@ -103,7 +103,7 @@ private Datum parseRecord(CSVRecord record) throws NumberFormatException {
                     badRows++;
                 }
             }
-            log.info("{}/{} rows successfully parsed ({} malformed rows)", numRows-badRows, numRows, badRows);
```

Comment:
lol nice catch. Was this actually causing issues?

Reply:
Not really. I just noticed that the line count numbers were wrong.